### ISSUE TITLE
ImportError: cannot import name descriptor

### ISSUE THREAD

**Issue by wanjianwei** [tensorflow/tensorflow#5858]
I installed tensorflow from source follow[ tutorial](https://www.tensorflow.org/versions/master/get_started/os_setup.html) and the os is macOS Sierra,tensorflow version is v0.11.0, i set up TensorFlow such that all files are linked (instead of copied) from the system directories, run the following commands inside the TensorFlow root directory:

        bazel build -c opt //tensorflow/tools/pip_package:build_pip_package_
        mkdir _python_build
        cd _python_build
        ln -s ../bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/org_tensorflow/* .
        ln -s ../tensorflow/tools/pip_package/* .
        python setup.py develop

when i follow the tutorial to retrain Inception's Final Layer for classifying flowers with  scripts as follows:

        “bazel build tensorflow/examples/image_retraining:retrain“
        “bazel-bin/tensorflow/examples/image_retraining/retrain --image_dir ~/flower_photos“

**i got the same error as follows:**

         Traceback (most recent call last):
         File "/Users/jway/Documents/tensorflow/bazel-       bin/tensorflow/examples/image_retraining/retrain.runfiles/org_tensorflow/tensorflow/examples/image_retraining/retrain.py", line 79, in <module>
         import tensorflow as tf
         File "/Users/jway/Documents/tensorflow/bazel-bin/tensorflow/examples/image_retraining/retrain.runfiles/org_tensorflow/tensorflow/__init__.py", line 24, in <module>
         from tensorflow.python import *
         File "/Users/jway/Documents/tensorflow/bazel-bin/tensorflow/examples/image_retraining/retrain.runfiles/org_tensorflow/tensorflow/python/__init__.py", line 63, in <module>
         from tensorflow.core.framework.graph_pb2 import *
         File "/Users/jway/Documents/tensorflow/bazel-bin/tensorflow/examples/image_retraining/retrain.runfiles/org_tensorflow/tensorflow/core/framework/graph_pb2.py", line 6, in <module>
         from google.protobuf import descriptor as _descriptor
         ImportError: cannot import name descriptor

I don't know how to solve it,can somebody give me some advices?

**Comment by prb12** [comment]
Please can you provide the information requested in the issues reporting template. 

It seems likely that you have an installation issue (protobuf library?) but it's impossible to triage this without OS and tensorflow versions, installation method, and the command lines needed to reproduce.

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity. When further information is posted we will reopen this. Thanks.

**Comment by sveinbjornt** [comment]
I'm having the same problem. Tensorflow was installed with pip on macOS Sierra.

**Comment by gautam1858** [comment]
Me too

**Comment by FruitcakefromMars** [comment]
I think I found a solution for this. Mine is on Windows however and I made my error "go away". I don't know if it is actually solved.

For other OS besides Windows, try:
`pip uninstall protobuf`
`brew install protobuf`
`mkdir -p /Library/Python/2.7/lib/python/site-packages`
`echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> /Library/Python/2.7/lib/python/site-packages/homebrew.pth
`

On Windows for me I uninstalled protobuf and tensorflow and reinstalled tensorflow only. "brew" doesn't work on cmd.
